Compass/CompassViewController.swift

```diff
@@ -10,59 +10,86 @@ import UIKit
 import CoreLocation
 import Parse
 
-class ViewController: UIViewController, CLLocationManagerDelegate {
+class CompassViewController: UIViewController, CLLocationManagerDelegate {
 
     var locationManager = CLLocationManager()
-    var currentLocation: CLLocation? = nil
-    var targetLocation: CLLocation? = nil
-    let par = PServer() 
+    var currentLocation: CLLocation?
+    var targetLocation: CLLocation?
+    let par = PServer()
     let myCalc = Calculations()
     var myUserData = UserData()
     
     let ArrowImage = UIImageView(image: UIImage(named: "240px-Green_Arrow_Up_Darker.png"))
     let degreeLabel = UILabel(frame: CGRect.zero)
+    let distanceLabel = UILabel(frame: CGRect.zero)
     let setTargetLocationButton = UIButton(frame: CGRect.zero)
+    let backButton = UIButton(frame: CGRect.zero)
     
     
     func initLocationManager() {
-        
-        //this function must be moved somewhere else, call once when app started
-        //par.initParse(appID: "492795c6ea25112881915677092fb19d95f43ce0", clKey: "6c4448eb0dc5d344a0ca35f8d8f978ff82b76028", serverAddress: "http://18.188.82.67:80/parse")
-        
-        if let tempID = UserDefaults.standard.string(forKey: "parseObjectID") {
-            myUserData.objectID = tempID
-        } else { myUserData.objectID = "" }
+        //Fault???
         if let userName = UserDefaults.standard.string(forKey: "userName") {
             myUserData.name = userName
-        } else { myUserData.objectID = "" }
-        
-        
+        } else { myUserData.name = ""
```

Comment:
Not a big deal, but try to keep the formatting consistent, it's something that I think people overvalue in interviews (probably because it's easy to spot, even though it's not a big deal).
There's absolutely nothing wrong with putting `myUserData.name = ""` on the same line as `else {` but it is something people tend to have strong opinions about, and most swift style guides would suggest doing the following, even when there's only one line in the else:
```
} else { 
    myUserData.name = ""
}